doc/classes/ShaderGlobalsOverride.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ShaderGlobalsOverride" inherits="Node" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Overrides global shader parameters' values in a specific scene.
+		A node used to override shader parameters' values in a scene.
```

Comment:
```suggestion
		A node used to override global shader parameters' values in a scene.
```

Global shader parameters are distinct from regular shader parameter. This only overrides global parameters